### PR TITLE
Setup core for unified pipelines.

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR sets up the core services directory for unified pipelines. Do we just need to ship the azure-core package from this pipeline or do we need to support shipping others?

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/core/ci.yml [hunk 1]
```diff
@@ -21,14 +35,10 @@ pr:
...
-jobs:
-- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
-    ServiceDirectory: core
\ No newline at end of file
+    ServiceDirectory: core
+    Artifacts:
+    - name: azure_core
```
Comment (weshaggard) [inline sdk/core/ci.yml]: Are we sure this is the only package we want to release with this pipeline?

Comment (mitchdenny) [reply sdk/core/ci.yml]: Yeah - I'm not sure ... adding @lmazuel and @xiangyan99 for confirmation.

Comment (lmazuel) [reply sdk/core/ci.yml]: we want to release azure-core-tracing-opencensus as well

Comment (mitchdenny) [reply sdk/core/ci.yml]: I have added azure-core-tracing-opencensus to the package list for the core pipeline. We can release them together or independently - it doesn't matter. Ideally everything produced by a pipeline would go out together but we've built this flexibility in.

**Pull request merged**